src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -21,27 +21,26 @@ dotnet add package OpenTelemetry.Exporter.OpenTelemetryProtocol
 You can configure the `OtlpExporter` through `OtlpExporterOptions` properties:
 
 * `Endpoint`: Target to which the exporter is going to send traces or metrics.
-The endpoint must be a valid Uri with scheme (http) and host, and
-MAY contain a port and path. In 1.0, secure connection(https) is *NOT*
-supported.
+  The endpoint must be a valid Uri with scheme (http) and host, and MAY contain
+  a port and path. In 1.0, secure connection(https) is *NOT* supported.
```

Comment:
we actually do support https, right! so this doc was never updated to reflect it..